state/protocol/badger/snapshot_test.go

```diff
@@ -196,9 +196,9 @@ func TestSealingSegment(t *testing.T) {
 			actual, err := state.AtBlockID(head.ID()).SealingSegment()
 			require.NoError(t, err)
 
-			assert.Len(t, actual, 1)
-			assert.Equal(t, len(expected), len(actual))
-			assert.Equal(t, expected[0].ID(), actual[0].ID())
+			assert.Len(t, actual.Blocks, 1)
+			assert.Equal(t, len(expected.Blocks), len(actual.Blocks))
+			assert.Equal(t, expected.Blocks[0].ID(), actual.Blocks[0].ID())
 		})
```

Comment:
better to add assertion on result list

Reply:
https://github.com/onflow/flow-go/pull/1533/commits/00a77e58055c9b7fbc3a8d569aa91866610f5646